doc/getting-started.md

```diff
@@ -414,6 +414,12 @@ Note that the model for MD simulations is required to be compatible with the Dee
 
 ### Run MD with LAMMPS
 
+Firstly, enable DeePMD-kit package in LAMMPS with `plugin` command:
+
+```
+plugin load path/to/deepmd/lib/libdeepmd_lmp.so
+```
+
```

Comment:
It seems that the built-in mode is not supported by this PR. Then it is a breaking change, because all lammps input scripts have to be changed. 

It is ideal to support both built-in and plugin modes. We still need the built-in mode because it is easy to use, one do not need to load the shared lib in every lammps script.

Reply:
Ok. Note that I just removed built-in mode in the document, and I'll revert them.